tests/Feature/Http/Controllers/EntryControllerTest.php

```diff
@@ -674,15 +674,36 @@ public function update_campaign_reward_clears_items_and_levels()
      */
     public function destroy_deletes_and_redirects()
     {
-        $entry = Entry::factory()->create();
+        $entry = Entry::factory()->create(['user_id' => $this->user->id]);
+        $user = $entry->user;
```

Comment:
isn't this the same as just doing
`$this->user`? since thats what youre passing into the entry creation.

Reply:
Good point. I believe I added 677 later on and didn't bother to refactor. Will do.